### PR TITLE
Added purge action

### PR DESCRIPTION
**Pull request by fgimenez**
This action can be used when deploying a new jenkins: because we are using instances from images created with udf called with --developer-mode, we are not using cloud-init nor the nova keypairs, the access is granted by udf copying the keys of the user creating the image to the image itself; when we create a new jenkins, new keys are created and the old images are no longer accessible.

After deploying the new jenkins and purging the old images, the image creation jobs take care of creating the new ones accessible with the new keys.

### CODE REVIEW

File: pkg/cloud/cloud.go [hunk 1]
```diff
@@ -141,7 +151,7 @@ func GetImageID(release, channel, arch string, version int) (name string) {
 
 // Delete calls the cli command to remove the given images
 func (c *Client) Delete(images ...string) (err error) {
-	_, err = c.cli.ExecCommand(append([]string{"openstack", "delete"}, images...)...)
+	_, err = c.cli.ExecCommand(append([]string{"openstack", "image", "delete"}, images...)...)
```
Comment (come-maiz) [inline pkg/cloud/cloud.go:154]: oops :)

**Comment by come-maiz** [comment]
This looks good, but I'm sorry, my nova reviews are still shots in the dark. I will use the time to enable snapcraft and bbb tests to also learn a little more about this.

File: pkg/cloud/cloud.go (hunk 1 above)
Comment (fgimenez) [inline pkg/cloud/cloud.go:154]: openstack delete yourself! :D

**Comment by fgimenez** [comment]
Thanks @elopio, merging

**Pull request merged**